it/uk/gov/hmrc/agentaccesscontrol/connectors/mtd/RelationshipsConnectorISpec.scala

```diff
@@ -14,52 +15,60 @@ class RelationshipsConnectorISpec extends WireMockWithOneAppPerSuiteISpec with M
   val client = MtdItId("C1111C")
```

Comment:
No longer much need for this definition at the top level - it's only referenced in one place now.